### ISSUE TITLE
仔细对比发现文章中微调后效果反而变差

### ISSUE THREAD

**Issue by lovean1** [DQiaole/MemFlow#18]
作者您好，复现对比结果时发现，原论文中模型在chairs和things上训练后在sintel和kitti上的性能很好，但是在针对微调后，性能反而有所下降，这是一个比较反直觉的现象，请问这一情况是由什么导致的，十分期待您的见解

**Comment by lovean1** [comment]
<img width="686" height="631" alt="Image" src="https://github.com/user-attachments/assets/2271e0c8-8521-48c6-93cc-7cac3647efcb" />

**Comment by lovean1** [comment]
<img width="1535" height="706" alt="Image" src="https://github.com/user-attachments/assets/651f4409-7baa-4fb8-8b5d-24e8e13d7d13" />

**Comment by DQiaole** [comment]
你好，仔细对比可以发现，其他方法也存在类似的现象。我猜测这是由于sintel数据集划分导致的。sintel原文提到：“The test set was chosen to be suﬃciently challenging, yet maintain similar image and flow statistics to the training set.”这句话似乎暗示测试集难度会更高。此外，sintel测试的时候还会对结果做稀疏化下采样，如果下采样过程中针对运动边缘做了特殊处理的话可能也会是测试结果偏高（这部分可能要看下采样的binary files的源代码）。
